mdesnoyer/wonderland
# Paid User Account Page #1285

# Changes
- updated `Account` to change content based on user type
- fully restyled `UserSettingsPage`
  - added `UserSettingsInfo`
  - restyled `UserSettingsForm`
  - removed `UserSettingsTab1` and `UserSettingsTab2`
- fully restyled `TelemetryPage` and `Support`
- partially restyled `Integrations`
- added css classes for `code` and `pre`
- added `_tabs.scss` for tab styling
- replaced Support Overview images
# Test Plan
- try signing into a normal, unpaid account
  - check that the "Account" tab in the sidebar does not include a "Settings" section
- try signing into a paid account
  - check that the "Account" tab in the sidebar has a "Settings" section that includes: User Settings, Billing, Plugins, Telemetry, Support, API Documentation
# Notes
- restyling `Billing` and `Integration` will be moved in another ticket
- Support content will need to be changed
